tests/common/plugins/pdu_controller/snmp_pdu_controllers.py

```diff
@@ -12,6 +12,7 @@
 
 logger = logging.getLogger(__name__)
 
+
```

Comment:
Why add this extra empty line?

Reply:
@Blueve Because [flake8 rule E302](https://www.flake8rules.com/rules/E302.html) requires two blank lines between classes.